includes/admin/class-wcs-admin-post-types.php

```diff
@@ -282,13 +282,33 @@ public function filter_bulk_actions( $actions ) {
 			'mark_completed',
 			'mark_cancelled',
 			'remove_personal_data',
+			'trash',
+			'delete',
+			'untrash',
 		];
 
 		// Remove actions that are not relevant to subscriptions.
 		$actions = array_diff_key( $actions, array_flip( $actions_to_remove ) );
 
-		// If we are currently in trash, expired or cancelled listing. We don't need to add subscriptions specific actions.
-		if ( in_array( $post_status, [ 'cancelled', 'trash', 'wc-expired' ], true ) ) {
+		// If we are currently in trash, expired or cancelled listing. We only need to add specific subscriptions actions.
```

Comment:
```suggestion
		// If we are currently in expired or cancelled listing. We only need to add specific subscriptions actions.
```

I suspect the `trash` reference was a typo?